### ISSUE TITLE
The add on is simply not working

### ISSUE THREAD

**Issue by AngeltheTraveller** [Liresol/anki-custom-shortcuts#154]
I go edit the commands in config but in actuality nothing happens. The shortcuts are just the same. And alt close still got 4 buttons

**Comment by Liresol** [comment]
Hello,
To make sure, have you restarted Anki before checking the settings? Addons don't restart until Anki restarts.
If that doesn't work, could you post which settings you have changed? It might help to see what settings you're trying to use in order to figure out what is going wrong.

**Comment by AngeltheTraveller** [comment]
Yes, I did do restart ANKI twice, then reinstalled the addon again and again. Restarted the PC still to no avail. The commands I am trying to change are the editor cloze to "Ctrl+Shift+D" and cloze alt to "Ctrl+Shift+C". And even though they are so written in the confing of the addon, if I try using em they are still by their default "Ctrl+Shift+C" and "Ctrl+Shift+alt+C". Also tried to set the editor cloze alt to <nop> and then use the no increment, but the alt one will continue to work as normal and the other one still wont exist

**Comment by Liresol** [comment]
That is pretty weird behavior. Do you have other addons that deal with clozes? That might be affecting how this addon deals with shortcuts. 
If you don't, could you test other shortcuts (like the reviewer shortcuts) to see if those are also broken?

**Comment by AngeltheTraveller** [comment]
Changing reviewer commands works properly. Also, I found a reviewer command that was using Ctrl+Shift+D and changed that. Now Ctrl+Shift+D does work on the normal cloze. However, Ctrl+Shift+C does not work yet on alt cloze (which for the love of god is all I want) and I tried to change the alt cloze on Ctrl+Shift+D but is still doing the regular add. And now both Ctrl+Shift+C and Ctrl+Shift+D
